overview/skills/public.mdx

```diff
@@ -1,27 +1,27 @@
 ---
-title: Global Microagents
-description: Global microagents are [keyword-triggered microagents](/openhands/usage/microagents/microagents-keyword) that apply to all OpenHands users. A list of the current global microagents can be found [in the OpenHands repository](https://github.com/OpenHands/OpenHands/tree/main/microagents).
+title: Global Skills (formerly Microagents)
+description: Global skills (formerly known as microagents) are [keyword-triggered skills](/overview/skills/keyword) that apply to all OpenHands users. A list of the current global skills can be found [in the OpenHands repository](https://github.com/OpenHands/OpenHands/tree/main/microagents).
 ---
 
-## Contributing a Global Microagent
+## Contributing a Global Skill
 
-You can create global microagents and share with the community by opening a pull request to the official repository.
+You can create global skills and share with the community by opening a pull request to the official repository.
 
 See the [CONTRIBUTING.md](https://github.com/OpenHands/OpenHands/blob/main/CONTRIBUTING.md) for specific instructions on how to contribute to OpenHands.
```

Comment:
@OpenHands update this and point to https://github.com/OpenHands/skills as the official global skill registry. Also mention in the overview page that we have a skill registry at https://github.com/OpenHands/skills